### DISCUSSION TITLE
Floating Window Font Size

### DISCUSSION THREAD

**Discussion by magerooster** [xorus/EngageTimer#71] [Ideas]
Would it be possible to get an option to change the size of the numbers for the floating window? The default size is unobtrusive, but I would like to make it more visible.

**Comment by xorus** [comment]
Unless I misunderstood what you're saying, you can already change it the configuration > floating window > styling. You can change the font size directly.
![image](https://github.com/user-attachments/assets/2a4dfe19-b6a1-4c5e-a104-fe88c65215d5)

**Comment by magerooster** [comment]
Oh, I see it now. I didn't notice Styling expanded out with more options. Thanks.
